certified-connectors/Yarado/apiDefinition.swagger.json

```diff
@@ -0,0 +1,165 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Yarado",
+    "description": "Yarado connector lets users list down robots and tasks belonging to their organisation and schedule a single task run.",
+	"contact": {
+	  "name": "Yarado Customer Service Center",
+	  "url": "https://yarado.atlassian.net/servicedesk/customer/portals",
+	  "email": "info@yarado.com"
+	}  
+  },
+  "host": "api.yarado.com",
+  "basePath": "/",
+  "x-ms-connector-metadata": [
+    {
+	  "propertyName": "Website",
+	  "propertyValue": "https://yarado.com"
+	},
+	{
+	  "propertyName": "Privacy policy",
+	  "propertyValue": "https://www.yarado.com/terms"
+	}, 
+	{
+	  "propertyName": "Categories",
+	  "propertyValue": "Business Management;Productivity"
+	}
+  ],
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/v1/robots": {
+      "get": {
+        "summary": "GET Robots",
+        "description": "GET Robots",
+        "operationId": "GetRobots",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "default",
```

Comment:
@MarcinKowalczukYarado please change it to OK so that it reflects that the operation was successful.